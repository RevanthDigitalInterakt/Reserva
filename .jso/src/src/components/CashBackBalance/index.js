  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  exports.default = CardCashback;
  var _asyncToGenerator2 = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[1]));
  var _slicedToArray2 = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[2]));
  var _react = _interopRequireWildcard(_$$_REQUIRE(_dependencyMap[3]));
  var _reactNative = _$$_REQUIRE(_dependencyMap[4]);
  var _IconWoodpecker = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[5]));
  var _EventProvider = _$$_REQUIRE(_dependencyMap[0])(_$$_REQUIRE(_dependencyMap[6]));
  function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(e) { return e ? t : r; })(e); }
  function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
  function CardCashback() {
    var navigation = (0, _$$_REQUIRE(_dependencyMap[7]).useNavigation)();
    var _useState = (0, _react.useState)(0),
      _useState2 = (0, _slicedToArray2.default)(_useState, 2),
      balance = _useState2[0],
      setBalance = _useState2[1];
    var _useAuthStore = (0, _$$_REQUIRE(_dependencyMap[8]).useAuthStore)(['profile']),
      profile = _useAuthStore.profile;
    var _useCashbackLazyQuery = (0, _$$_REQUIRE(_dependencyMap[9]).useCashbackLazyQuery)({
        context: {
          clientName: 'gateway'
        },
        fetchPolicy: 'cache-and-network'
      }),
      _useCashbackLazyQuery2 = (0, _slicedToArray2.default)(_useCashbackLazyQuery, 1),
      getCashback = _useCashbackLazyQuery2[0];
    var getCashbackData = (0, _react.useCallback)(/*#__PURE__*/(0, _asyncToGenerator2.default)(function* () {
      try {
        var _yield$getCashback = yield getCashback(),
          data = _yield$getCashback.data;
        if (!(data != null && data.cashback)) return;
        var wallet = data.cashback.wallet;
        setBalance(wallet == null ? undefined : wallet.balanceInCents);
      } catch (error) {
        _$$_REQUIRE(_dependencyMap[10]).ExceptionProvider.captureException(error, {
          currentProfileDocument: profile == null ? undefined : profile.document
        });
      }
    }), [getCashback, profile]);
    (0, _react.useEffect)(function () {
      if (profile != null && profile.document) {
        getCashbackData();
      }
    }, [profile == null ? undefined : profile.document]);
    var handleClick = function handleClick() {
      _EventProvider.default.logEvent('click_card_cashback', {
        value: balance
      });
      navigation.navigate(_$$_REQUIRE(_dependencyMap[11]).MyCashbackScreensRoutes.MY_WALLET);
    };
    return /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[12]).jsxs)(_reactNative.TouchableOpacity, {
      style: _$$_REQUIRE(_dependencyMap[13]).styles.card,
      onPress: handleClick,
      children: [/*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[12]).jsxs)(_reactNative.View, {
        style: _$$_REQUIRE(_dependencyMap[13]).styles.textsContainer,
        children: [/*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[12]).jsx)(_reactNative.Text, {
          style: _$$_REQUIRE(_dependencyMap[13]).styles.title,
          children: "Saldo Cashback"
        }), /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[12]).jsx)(_$$_REQUIRE(_dependencyMap[14]).PriceCustom, {
          fontFamily: _$$_REQUIRE(_dependencyMap[15]).FONTS.RESERVA_SANS_MEDIUM,
          sizeInteger: 12,
          sizeDecimal: 10,
          num: balance || 0,
          color: "white"
        })]
      }), /*#__PURE__*/(0, _$$_REQUIRE(_dependencyMap[12]).jsx)(_IconWoodpecker.default, {})]
    });
  }
