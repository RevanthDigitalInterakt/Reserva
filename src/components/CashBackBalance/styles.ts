import { StyleSheet } from 'react-native';
import { FONTS } from '../../base/styles';

export const styles = StyleSheet.create({
  card: {
    height: 42,
    width: 179,
    backgroundColor: '#000',
    borderRadius: 4,
    alignItems: 'center',
    flexDirection: 'row',
    padding: 10,
  },
  title: {
    color: '#F8CD78',
    fontFamily: FONTS.RESERVA_SANS_REGULAR,
  },
  textsContainer: {
    marginRight: 10,
  },
  amount: {
    color: '#fff',
    fontWeight: '700',
    fontFamily: FONTS.RESERVA_SANS_MEDIUM,
  },
});
