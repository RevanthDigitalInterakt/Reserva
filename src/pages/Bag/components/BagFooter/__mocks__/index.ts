import { profileQuery } from '../../../../../graphql/profile/profileQuery';

export const bagInfos = {
  totalBagItemsPrice: 100,
  totalBagItems: 20,
  totalBagDiscountPrice: 40,
  totalBagDeliveryPrice: 60,
};

export const currentOrderForm: any = {
  items: [
    {
      name: 'Produto A',
      skuName: 'test',
      price: 50.0,
      quantity: 2,
      productId: 4,
      productCategories: [
        {},
      ],
      additionalInfo: {
        brandName: 'RESERVA',
      },
    },
    {
      name: 'Produto B',
      skuName: 'test',
      price: 20.0,
      quantity: 1,
      productId: 4,
      productCategories: [
        {},
      ],
      additionalInfo: {
        brandName: 'RESERVA',
      },
    },
  ],
  orderFormId: '1',
};

export const installmentInfo = {
  installmentPrice: 110,
  installmentsNumber: 40,
  totalPrice: 1000,
};

export const apolloMocks = [
  {
    request: {
      query: profileQuery,
      variables: {},
    },
    result: {
      data: {
        profile: {
          __typename:
            'vtex_storegraphql_2_160_0_Profile',
          addresses: [[{}]],
          customFields: [[{}], [{}], [{}]],
          userId: '4d35c3cf-d4a1-4c52-a421-66d8f97f1b10',
          firstName: 'Tester',
          lastName: 'Silva',
          fullName: 'teste da Silva',
          email: 'nogueirahy@gmail.com',
          document: '11111111111',
          birthDate: '1980-01-01T00:00:00.000Z',
          gender: 'male',
          homePhone: '+5511991111111',
          payments: null,
          passwordLastUpdate: 'teste123',
        },
      },
    },
  },
];

export const apolloMocksWithoutDataUser = [
  {
    request: {
      query: profileQuery,
      variables: {},
    },
    result: {
      data: {
        profile: {
          __typename:
            'vtex_storegraphql_2_160_0_Profile',
          userId: '4d35c3cf-d4a1-4c52-a421-66d8f97f1b10',
          firstName: null,
          lastName: null,
          fullName: null,
          email: 'nogueirahy@gmail.com',
          document: null,
          birthDate: null,
          gender: 'male',
          homePhone: null,
          payments: null,
          passwordLastUpdate: 'teste123',
        },
      },
    },
  },
];
