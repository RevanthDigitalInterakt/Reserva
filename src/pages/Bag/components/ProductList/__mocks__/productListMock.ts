import type { OrderformOutput } from '../../../../../base/graphql/generated';

export const orderFormMock = {
  orderFormId: '85b805c4c8c04d99b9cc0bf61d1c58e1',
  appTotalizers: {
    items: 828,
    discount: -79.71,
    delivery: 0,
    total: 748.29,
    __typename: 'OrderformAppTotalizersOutput',
  },
  items: [
    {
      productTitle: 'CAMISA ML SAMOA BRANCO',
      itemColor: 'BRANCO',
      itemSize: 'P',
      isGift: false,
      isGiftable: true,
      imageSource: 'https://lojausereserva.vteximg.com.br/arquivos/ids/6413946/0054128014_01.jpg?v=637692018478770000',
      key: '90368-39900-19929-19929-19929-1-1-camisa-ml-samoa-branco-p-',
      isAssinaturaSimples: false,
      priceWithDiscount: 199.29,
      discountPercent: 50,
      discountApi: 0,
      showFirstPurchaseDiscountMessage: null,
      showTotalDiscountFirstPurchaseValue: null,
      price: 27900,
      productId: '3239',
      id: '90368',
      listPrice: 39900,
      giftOfferingId: '90310',
      seller: '1',
      skuName: 'BRANCO - P',
      uniqueId: '52D9B57139FA4292909748AE8ED11774',
      isAddedAsGift: false,
      name: 'CAMISA ML SAMOA BRANCO - P',
      quantity: 1,
      disableCounter: false,
      sellingPrice: 19929,
      __typename: 'OrderformItemOutput',
    },
    {
      productTitle: 'TENIS RSV NEO RE BRANCO/VERDE MILITAR',
      itemColor: 'BRANCO/VERDE MILITAR',
      itemSize: '41',
      isGift: false,
      isGiftable: true,
      imageSource: 'https://lojausereserva.vteximg.com.br/arquivos/ids/7575487/0075733901_01.jpg?v=638145040244900000',
      key: '426190-54900-54900-54900-54900-lojausereservago-1-tenis-rsv-neo-re-branco-verde-militar-41-',
      isAssinaturaSimples: false,
      priceWithDiscount: 549,
      discountPercent: 0,
      discountApi: 0,
      showFirstPurchaseDiscountMessage: null,
      showTotalDiscountFirstPurchaseValue: null,
      price: 54900,
      productId: '40572',
      id: '426190',
      listPrice: 54900,
      giftOfferingId: '27623',
      seller: 'lojausereservago',
      skuName: 'BRANCO/VERDE MILITAR - 41',
      uniqueId: '49A868C5C6974936AD7BA870A830A7A8',
      isAddedAsGift: false,
      name: 'TENIS RSV NEO RE BRANCO/VERDE MILITAR - 41',
      quantity: 1,
      disableCounter: false,
      sellingPrice: 54900,
      __typename: 'OrderformItemOutput',
    },
  ],
  selectableGift: null,
  allItemsQuantity: 2,
  installmentInfo: {
    installmentsNumber: 12,
    installmentPrice: 62.35,
    totalPrice: 748.29,
    __typename: 'OrderformInstallmentInfoOutput',
  },
  marketingData: null,
  __typename: 'OrderformOutput',
} as OrderformOutput;
