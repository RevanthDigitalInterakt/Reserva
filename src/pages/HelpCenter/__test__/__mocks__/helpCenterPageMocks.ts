import type { DocumentNode } from 'graphql';
import { HelpCenterCollectionDocument, type HelpCenterCollectionQuery } from '../../../../base/graphql/generated';
import { mockHelpCenterCollectionData } from '../../../../zustand/useHelpCenterStore/mocks/helpCenterCollection.mock';

interface IApolloMock<T> {
  request: {
    query: DocumentNode,
    variables: object,
  },
  result: {
    data: T
  }
}

export const helpCenterPageMocks: Array<IApolloMock<HelpCenterCollectionQuery>> = [
  {
    request: {
      query: HelpCenterCollectionDocument,
      variables: {},
    },
    result: {
      data: {
        helpCenterCollection: {
          ...mockHelpCenterCollectionData,
        },
        __typename: 'Query',
      },
    },
  },
];
