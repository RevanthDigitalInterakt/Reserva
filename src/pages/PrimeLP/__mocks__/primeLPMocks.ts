import type { DocumentNode } from 'graphql';
import { LandingPagePrimeDocument, LandingPagePrimeQuery } from '../../../base/graphql/generated';
import { mockPrimeData } from '../../../../__mocks__/PrimeLP.mock';

interface IApolloMock<T> {
  request: {
    query: DocumentNode,
    variables: object,
  },
  result: {
    data: T
  }
}

export const ApolloMockLPPrime: Array<IApolloMock<LandingPagePrimeQuery>> = [
  {
    request: {
      query: LandingPagePrimeDocument,
      variables: {},
    },
    result: {
      data: {
        landingPagePrime: {
          ...mockPrimeData,
        },
        __typename: 'Query',
      },
    },
  },
];
