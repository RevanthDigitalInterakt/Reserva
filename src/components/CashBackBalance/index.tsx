import React, { useCallback, useEffect, useState } from 'react';
import { View, Text, TouchableOpacity } from 'react-native';
import { useNavigation } from '@react-navigation/native';
import IconWoodpecker from '../../../assets/icons/IconWoodpecker';
import { FONTS } from '../../base/styles';
import { useCashbackLazyQuery } from '../../base/graphql/generated';
import { ExceptionProvider } from '../../base/providers/ExceptionProvider';
import { useAuthStore } from '../../zustand/useAuth/useAuthStore';
import { PriceCustom } from '../PriceCustom/PriceCustom';
import { styles } from './styles';
import EventProvider from '../../utils/EventProvider';
import { MyCashbackScreensRoutes } from '../../modules/my-cashback/navigation/MyCashbackNavigator';

export default function CardCashback() {
  const navigation = useNavigation();
  const [balance, setBalance] = useState<number>(0);
  const { profile } = useAuthStore(['profile']);
  const [getCashback] = useCashbackLazyQuery({
    context: { clientName: 'gateway' }, fetchPolicy: 'cache-and-network',
  });
  const getCashbackData = useCallback(async () => {
    try {
      const { data } = await getCashback();

      if (!data?.cashback) return;
      const { wallet } = data.cashback;
      setBalance(wallet?.balanceInCents);
    } catch (error) {
      ExceptionProvider.captureException(error, { currentProfileDocument: profile?.document });
    }
  }, [getCashback, profile]);

  useEffect(() => {
    if (profile?.document) {
      getCashbackData();
    }
  }, [profile?.document]);

  const handleClick = () => {
    EventProvider.logEvent('click_card_cashback', {
      value: balance,
    });
    navigation.navigate(MyCashbackScreensRoutes.MY_WALLET);
  };

  return (
    <TouchableOpacity style={styles.card} onPress={handleClick}>
      <View style={styles.textsContainer}>
        <Text style={styles.title}>Saldo Cashback</Text>
        <PriceCustom
          fontFamily={FONTS.RESERVA_SANS_MEDIUM}
          sizeInteger={12}
          sizeDecimal={10}
          num={balance || 0}
          color="white"
        />
      </View>

      <IconWoodpecker />
    </TouchableOpacity>
  );
}
