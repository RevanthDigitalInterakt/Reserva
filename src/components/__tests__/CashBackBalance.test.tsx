import React from 'react';
import { render } from '@testing-library/react-native';
import CashBackBalance from '../CashBackBalance';
import { mockResponseCashBack } from '../../../__mocks__/mockResponseCashBack';

jest.mock('../../base/graphql/generated', () => ({
  useCashbackLazyQuery: () => [
    jest.fn(() => ({
      data: mockResponseCashBack,
    })),
  ],
}));

const Component = (
  <CashBackBalance />
);

describe('CashBackBalance', () => {
  it('should return match snapshot', () => {
    const { toJSON } = render(Component);

    expect(toJSON()).toMatchSnapshot();
  });
  // TODO more test cases
});
