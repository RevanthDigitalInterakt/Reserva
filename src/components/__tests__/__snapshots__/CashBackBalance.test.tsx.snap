// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CashBackBalance should return match snapshot 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#000",
      "borderRadius": 4,
      "flexDirection": "row",
      "height": 42,
      "opacity": 1,
      "padding": 10,
      "width": 179,
    }
  }
>
  <View
    style={
      {
        "marginRight": 10,
      }
    }
  >
    <Text
      style={
        {
          "color": "#F8CD78",
          "fontFamily": "ReservaSans-Regular",
        }
      }
    >
      Saldo Cashback
    </Text>
    <View
      style={
        {
          "flexDirection": "row",
        }
      }
    >
      <View>
        <View
          style={
            {
              "flexDirection": "row",
            }
          }
        >
          <Text
            style={
              {
                "color": "white",
                "fontFamily": "ReservaSans-Medium",
                "fontSize": 28.125,
                "lineHeight": undefined,
              }
            }
          >
            R$
             
          </Text>
          <Text
            style={
              {
                "color": "white",
                "fontFamily": "ReservaSans-Medium",
                "fontSize": 28.125,
                "lineHeight": undefined,
                "textDecorationLine": "none",
              }
            }
          >
            0
          </Text>
          <Text
            style={
              {
                "color": "white",
                "fontFamily": "ReservaSans-Medium",
                "fontSize": 28.125,
                "lineHeight": undefined,
              }
            }
          >
            ,
          </Text>
        </View>
      </View>
      <View
        style={
          {
            "alignSelf": "flex-start",
          }
        }
      >
        <Text
          style={
            {
              "color": "white",
              "fontFamily": "ReservaSans-Medium",
              "fontSize": 23.4375,
              "textDecorationLine": "none",
            }
          }
        >
          00
        </Text>
      </View>
    </View>
  </View>
  <RNSVGSvgView
    align="xMidYMid"
    bbHeight="33"
    bbWidth="28"
    fill="#1B170F"
    focusable={false}
    height="33"
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      [
        {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        {
          "flex": 0,
          "height": 33,
          "width": 28,
        },
      ]
    }
    vbHeight={33}
    vbWidth={28}
    width="28"
  >
    <RNSVGGroup
      fill={
        {
          "payload": 4279965455,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        d="M27.2057 30.8142C26.7057 30.428 24.4324 28.9451 23.3309 27.972C22.9012 27.5935 23.2059 27.1533 22.7293 26.6976C22.12 26.1184 21.4481 25.485 20.7841 24.8131C20.0888 24.1026 19.4326 23.3225 18.7998 22.5115C18.2217 21.716 16.4249 19.2677 14.964 17.6381C13.3938 15.8926 10.433 13.761 9.61269 11.8069C9.56582 11.6756 9.51895 11.5443 9.47988 11.4053C9.3002 10.7797 9.20645 10.0769 9.2377 9.25823C9.26113 8.47044 9.50331 7.66721 10.3861 6.33107C11.3548 4.87907 11.1829 4.23803 10.847 4.18397C10.5189 4.1299 10.1595 4.14535 10.5189 2.63157C10.8314 1.31859 10.6126 0.770234 10.4095 0.438128C10.222 0.129193 9.98767 -0.0329981 9.80799 0.00561876C9.39395 0.0905759 9.07364 1.06372 8.66741 1.31859C8.56585 1.38038 8.02682 1.68159 8.13619 0.453575C8.15963 0.160087 7.95651 0.0674059 7.68308 0.23732C7.26122 0.492192 6.66749 1.16413 6.34719 2.41531C6.32376 2.49255 6.25345 2.51572 6.21439 2.41531C6.1597 2.26085 6.14408 1.97508 6.26126 1.48078C6.37063 1.02511 6.56593 0.654383 6.34719 0.608043C6.08158 0.546256 5.69879 1.09462 5.40193 1.72793C5.35505 1.82061 5.34724 1.85151 5.31599 1.92874C5.28474 2.00597 5.19099 2.05231 5.19099 1.89012C5.19099 1.782 5.21443 1.60436 5.27693 1.3881C5.30036 1.29542 5.25349 1.25681 5.21443 1.24908C5.1285 1.22591 5.01913 1.27225 4.88632 1.37266C4.58164 1.59664 4.29259 1.99825 4.08167 2.53889C4.05823 2.59295 4.0426 2.65474 4.01917 2.71653C4.01917 2.73197 4.01135 2.7397 4.00354 2.75514C4.00354 2.77059 3.98792 2.79376 3.98792 2.80921C3.98011 2.8401 3.97229 2.87099 3.96448 2.90189C3.94885 2.9405 3.94105 2.98684 3.93323 3.02546C3.93323 3.04091 3.93324 3.05635 3.92543 3.0718C3.9098 3.14903 3.88636 3.23399 3.87073 3.31895C3.85511 3.41163 3.83949 3.50431 3.82386 3.58927C3.82386 3.58927 3.82386 3.60471 3.82386 3.61244C3.82386 3.61244 3.82386 3.62016 3.82386 3.62788C3.79261 3.87503 3.7848 4.12218 3.7848 4.36161C3.7848 4.40022 3.7848 4.43884 3.7848 4.48518C3.7848 4.61648 3.47231 4.64737 3.54262 4.94086C3.71449 5.6823 3.99573 6.43919 4.14417 6.82536C4.19885 6.97211 4.12073 7.12657 3.9723 7.17291C3.2692 7.37372 1.4099 7.95298 0.0974502 8.74076C0.0193284 8.7871 -0.035354 8.91067 0.0271434 9.01108C0.0896409 9.10376 0.245881 9.09604 0.31619 9.06514C1.81613 8.57085 3.95667 7.88346 4.98788 8.61719C5.74566 9.15782 5.82378 10.4476 5.34724 11.8456C5.19099 12.363 5.01131 12.8264 4.84726 13.2976C2.65203 19.0437 6.69874 21.6929 10.4252 25.0989C10.6986 25.346 11.2689 25.9793 11.7688 26.7131C12.136 27.246 12.5032 27.833 12.4641 28.4122C12.4329 28.8988 12.3548 29.4858 12.1829 30.2426C11.972 31.1695 11.7454 30.7138 10.2689 30.3431C9.80799 30.2272 9.78456 30.2658 9.48769 30.8914C9.18302 31.5325 9.15958 31.6097 9.2377 31.6483C9.30019 31.6792 12.0735 31.6637 12.386 31.7255C12.4329 31.7333 12.4798 31.7487 12.5266 31.7719C12.8626 31.9263 14.3859 32.6523 14.8547 32.9613C14.9562 33.0308 15.1047 32.9999 15.1671 32.9227C15.2375 32.8223 15.1828 32.6987 15.0968 32.6446C14.4094 32.2276 13.0266 31.1926 12.9875 30.876C12.9719 30.7524 13.0891 30.289 13.1438 29.9337C13.3938 28.3427 13.4328 27.7403 14.0891 27.4391C14.6984 27.1533 16.128 27.8793 17.253 28.1496C17.2999 28.1651 17.3389 28.2114 17.3233 28.2577C17.2842 28.4199 17.1983 28.7907 17.003 29.424C16.9796 29.5089 16.9093 29.7175 16.7218 29.7715C16.4796 29.841 15.5421 29.4935 14.9093 29.3467C14.6437 29.285 14.5656 29.115 13.9641 30.3662C13.925 30.4512 13.9406 30.4898 13.9641 30.513C14.0031 30.5439 16.6358 30.6674 17.0108 30.7447C17.0811 30.7601 17.1436 30.7756 17.2061 30.8065C18.1983 31.2853 18.9951 31.7178 19.5263 32.0036C19.8154 32.158 19.9091 32.2276 20.0107 32.1272C20.1435 31.9959 20.0029 31.88 19.9013 31.7873C19.8466 31.7333 19.2451 31.3162 18.6748 30.9146C18.128 30.5284 17.5967 30.2813 17.6124 30.0805C17.6202 29.9723 17.6905 29.6557 17.7139 29.5553C17.7295 29.4858 17.9327 28.7598 18.003 28.4894C18.0186 28.4354 18.0733 28.4045 18.1202 28.4277C20.9482 29.3699 21.8153 31.1772 23.9715 32.2971C26.5573 33.6409 29.4166 32.5365 27.2135 30.8219L27.2057 30.8142Z"
        fill={
          {
            "payload": 4279965455,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
